### PR TITLE
[OKEX] support GZIP compression

### PR DESCRIPTION
**Pull request by declan94**
Related to issue #238

### CODE REVIEW

**Comment by amit2103** [comment]
Guys is there any plan to merge this ? Because I see that Okex will stop support after 24th this month

**Comment by amit2103** [comment]
Guys is there any plan to merge this ? Because I see that Okex will stop support after 24th this month

**Comment by declan94** [comment]
> Guys is there any plan to merge this ? Because I see that Okex will stop support after 24th this month

Yes, that's true. So I've merged this PR in my own repo.

**Comment by declan94** [comment]
Oh, they have changed the due date to Oct. 30.

**Comment by amit2103** [comment]
Okay, yea seems so, but anyways if this is reviewed and merged quickly thats better.

**Comment by Flemingjp** [comment]
@amit2103 Add your review of the code after testing this PR and I will merge it

**Comment by amit2103** [comment]
I have started testing it

**Comment by amit2103** [comment]
There is one minor observation. I have tested the code , it seems to work fine.

**Comment by amit2103** [comment]
@Flemingjp  I have added my review and tested it. Please let me know if you need anything else

**Comment by Flemingjp** [comment]
@declan94 If you could follow up the review, ill go from there

**Comment by declan94** [comment]
Sure, of course

File: service-netty/src/main/java/info/bitrich/xchangestream/service/netty/WebSocketClientHandler.java [hunk 1]
```diff
@@ -25,8 +25,8 @@
         public void onMessage(String message);
     }
 
-    private final WebSocketClientHandshaker handshaker;
-    private final WebSocketMessageHandler handler;
+    protected final WebSocketClientHandshaker handshaker;
```
[the remark this answers is not in the document]
Comment (declan94) [reply service-netty/src/main/java/info/bitrich/xchangestream/service/netty/WebSocketClientHandler.java]: The fields are both "final" fields, so it seems no risk to exposing the fields as protected fields.

[the remark this answers is not in the document]
Comment (amit2103) [reply service-netty/src/main/java/info/bitrich/xchangestream/service/netty/WebSocketClientHandler.java]: Yea I know but still, getters should be. good enough right ?

**Comment by amit2103** [comment]
Gusy any progress on this ? @Flemingjp  can you please take a look

**Comment by Flemingjp** [comment]
@amit2103 Sorry, I was still waiting on your approving review - have you guys resolved the protected final field discussion?

**Comment by amit2103** [comment]
actually its fine if getter is used, its not a blocker

**Pull request merged**